api/inventorypb/agents.proto

```diff
@@ -682,6 +682,10 @@ message AddMongoDBExporterRequest {
   bool push_metrics = 9;
   // List of collector names to disable in this exporter.
   repeated string disable_collectors = 13;
+  // Authentication mechanism.
+  // See https://docs.mongodb.com/manual/reference/parameters/#mongodb-parameter-param.authenticationMechanisms
```

Comment:
shouldn't it be `https://docs.mongodb.com/manual/reference/connection-string/#mongodb-urioption-urioption.authMechanism`?